warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -260,11 +260,21 @@ public int run(String[] args) throws Exception {
         log.setLevel(Level.INFO);
         
         // Initialize the markings file helper so we get the right markings file
+        long start = System.currentTimeMillis();
```

Comment:
This would be a great place to look at the org.slf4j.profiler.StopWatch class. I would much prefer that (one stopwatch with named timers) to the repetitive checking of current time.